web/src/helpers/formatting.js

```diff
@@ -2,6 +2,7 @@
 // TODO: remove once refactored
```

Comment:
Ooooh, just answering my own question below about why linting is not working 😄 
Can you remove the `/* eslint-dsiable */` comment and apply linting? :)

Reply:
Seems a sea monster has been hiding here

<img width="778" alt="image" src="https://user-images.githubusercontent.com/45588799/171822506-e59ad249-49aa-4151-8b40-2d13b8a6ddef.png">

Reply:
Haha yeah, ouch 😅